scrapy/extensions/robotstxtparser.py

```diff
@@ -0,0 +1,137 @@
+import sys
+import logging
+
+from scrapy.utils.python import to_native_str, to_unicode
+
+logger = logging.getLogger(__name__)
+
+class PythonRobotParser():
```

Comment:
it should be either `PythonRobotParser:` (Python 3 style), or `PythonRobotParser(object)` (careful Python 2 style)